src/GraphQL/Types/DateGraphType.cs

```diff
@@ -21,31 +21,31 @@ public override object Serialize(object value)
 
         public override object ParseValue(object value)
         {
-            string inputValue;
-            DateTime dateTime;
-
             if (value is DateTime)
             {
-                inputValue = ((DateTime)value).ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss.FFFFFFF'Z'");
-            }
-            else
-            {
-                inputValue = value != null ? value.ToString().Trim('"') : string.Empty;
+                return value;
```

Comment:
We perhaps should check to see if the `DateTime` is a UTC date and if not convert it before it is returned.